src/Action/AddAction.php

```diff
@@ -126,7 +126,7 @@ protected function _post()
             return $this->_success($subject);
         }
 
-        return $this->_error($subject);
+        $this->_error($subject);
```

Comment:
Damn this actually breaks a change I have where I redirect back to a different page on error.

Any thoughts on how to do that with this change?